### PR TITLE
all done

### PR DESCRIPTION
**Pull request by AndrZhuk**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -1 +1,65 @@
...
+        if self.action == "list":
+            return MovieListSerializer
+        if self.action == "retrieve":
+            return MovieRetrieveSerializer
+        return MovieSerializer
+
+    def retrieve(self, request, *args, **kwargs):
+        instance = self.get_object()
+        serializer = self.get_serializer(instance)
+        print(type(serializer.data))
```
Comment (vsmutok) [inline cinema/views.py]: remove this `print`

File: cinema/urls.py [hunk 2]
```diff
@@ -1 +1,28 @@
...
+router.register("movies", MovieViewSet, basename="movies")
+router.register(
+    "movie_sessions",
+    MovieSessionViewSet,
+    basename="movie_sessions"
+)
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```
Comment (vsmutok) [inline cinema/urls.py]: You could simplify it to urlpatterns = router.urls

File: cinema/models.py [hunk 3]
```diff
@@ -27,6 +27,10 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
+    def full_name(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
     def __str__(self):
         return self.first_name + " " + self.last_name
```
Comment (vsmutok) [inline cinema/models.py]: You are setting full_name as a property and then getting it by calling str?
Only 1 of those actions should be enough.

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: cinema/models.py (hunk 3 above)
Comment (vsmutok) [inline cinema/models.py]: use  `full_name` here

File: cinema_service/settings.py [hunk 4]
```diff
@@ -20,9 +20,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
+SECRET_KEY = \
```
Comment (vsmutok) [inline cinema_service/settings.py]: better to use `()` then `\`

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: cinema/serializers.py [hunk 5]
```diff
@@ -1 +1,130 @@
...
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["name", "rows", "seats_in_row", "capacity"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = "__all__"
```
Comment (Dimosphen1) [inline cinema/serializers.py:38]: Since `__all__` is a default value, you could not include it in the future

**Review by Dimosphen1 — APPROVED** [review]
LGTM, well done! One comment is left for consideration

**Pull request closed without merging**